server/src/routes/clientRouter.js

```diff
@@ -1,11 +1,18 @@
-import express from "express"
-import getClientIndexPath from "../config/getClientIndexPath.js"
+import express from "express";
+import getClientIndexPath from "../config/getClientIndexPath.js";
 
-const router = new express.Router()
+const router = new express.Router();
 
-const clientRoutes = ["/", "/user-sessions/new", "/users/new", "/podcasts", '/genres']
+const clientRoutes = [
+  "/",
+  "/user-sessions/new",
+  "/users/new",
+  "/podcasts",
+  "/genres",
+  "podcasts/:id",
+];
 router.get(clientRoutes, (req, res) => {
-  res.sendFile(getClientIndexPath())
-})
+  res.sendFile(getClientIndexPath());
+});
 
-export default router
+export default router;
```

Comment:
Same semicolon vs no semi colon debate.  You can change the `.prettierrc` file settings.  My guess is some members have `"semi": true` and some have `"semi": false`